libconnman-qt/networkmanager.cpp

```diff
@@ -1300,6 +1330,24 @@ void NetworkManager::propertyChanged(const QString &name, const QVariant &value)
 {
     if (name == State) {
         m_priv->updateState(value.toString());
+    } else if (name == DefaultService) {
+            NetworkService* newDefaultRoute(NULL);
+            QString path = value.toString();
+
+            qDebug() << "Default service changed to path \'" << path << "\'";
+
+            newDefaultRoute = selectDefaultRoute(path);
+
+            if (m_defaultRoute != newDefaultRoute || m_defaultRouteIsVPN) {
+                qDebug() << "Updating default route";
+                m_defaultRoute = newDefaultRoute;
+
+                /* Unset only when default is not VPN */
+                if (path.indexOf("vpn_") == -1)
+                    m_defaultRouteIsVPN = false;
+
+                updateDefaultRoute();
+            }
```

Comment:
This block has excessive indentation compared to surrounding code.

Reply:
Yeah, overlooked this part. Formatting fixed in d90b034ef0128e0196b1ca097354074b1e695b79